# Introduction

I will be reproducing the following figures from Kandeel, Mahmoud, Maged E. M. Mohamed, Hany M. Abd El-Lateef, Katharigatta N.
Venugopala, Hossam S. El-Beltagi, “Omicron variant genome evolution and phylogenetics,”
Medical Virology 2022; 94:1627-1632. To achieve this, I will use data on COVID genomes collected by my lab. 

# Figure to Reproduce

![Figures to Reproduce](https://i.imgur.com/OVo6aSL.png)

# Materials and Methods

I will be utilizing COVID-19 genomes which will be provided to me by my lab. I intend to also obtain genomes fro
geographic locations outside of North America to try and create some inferences as to the region of origin of the
genomes which were sequenced by my lab.

The phylogenetic trees will likely be created using Geneious software, which is also utilized by the authors of the
figure that I am trying to reproduce. My current goal is to utilize all of their methods, including the generation of
pairwise comparative matrices. To create the trees themselves, I intend to use the Neighbor Joining Method (NJM) and 
Kimura 80 to estimate evolutionary distance between the nodes. I would like to try and hand-calculate either UPGMA 
or NJM to gain a better understanding of how these simpler algorithms actually work.

If I find that the alignments and creation of the trees take shorter than expected, I will consider incorporating more
samples into my tree or using a more complicated model, such as a maximum likeliness method or a Bayesian method.
