# Response: Omicron variant genome evolution and phylogenetics
Tristan Szymanski
_______
## Introduction
An article published in the *Journal of Medical Virology* in 2022 by M. Kandeel, M.E.M. Mohamed, H.M Abd El-Lateef, K.N. 
Venugopala and H. S. El-Beltagi [1] sought to establish a putative phylogeny of SARS-CoV-2 variation which included the 
emerging Omicron variant. The authors utilized 8 SARS-CoV-2 genomes sourced from the GISAID Epi-CoV database. The authors 
performed phylogenetic analyses on these sequences using the UPGMA and Neighbor-Joining (NJ) clustering methods and the 
Jukes-Cantor and Kimura 80 (K80) substitution models. In total, 4 phylogenetic analyses were performed which used each 
clustering method and substitution model to determine a model of the evolutionary history of the Omicron variant. The 
authors generated two contrasting results, one which showed Omicron as highly distant when analyzed using either method of 
tree construction and the K80 substitution model; and one which showed Omicron as closely related to Alpha but still 
monophyletic when analyzed using the NJ method and the Jukes-Cantor.

This response was prompted after identifying a few elements of concern in the original paper which could be responsible for 
the conflicting evolutionary trees generated by Kandeel et al. First, the small sample size of SARS-CoV-2 genomes combined 
with the clustering methods utilized may have not been sufficient data for the phylogenetic methods to produce an accurate 
tree. Increasing the number of samples used could lead to different results. Second, there is a distinct lack of geographic 
diversity in the sequences the authors chose to utilize. Eliminating the GH490R variant from France resulted in a total of 5
of the 7 sequences being from Japan alone. It is possible that the relationships being shown are the result of higher gene 
flow between the Japanese sequences in comparison to the Botswana and American sequences. Finally, utilizing methods based on
Maximum-Likelihood with a more nuanced substitution model could capture greater information than the Jukes-Cantor (JC) or K80 
combined with UPGMA and NJM. This response seeks to address these concerns, to produce a new model of the evolutionary 
history of the Omicron variant with these concerns in mind, and to elaborate on the advantages of using Maximum Likelihood 
(ML) methods for answering questions regarding the evolutionary history of SARS-CoV-2 variant lineages.
___________
## Materials and Methods
________
### Reference Figures
|![Kandeel et al, 2022. Figure showing the accession information of genomes used.](https://i.imgur.com/w6khfc0.png)|
|:---:|
|**Figure 1:** Accession information of the sequences utilized by Kandeel et al., 2022.|
|![Kandeel et al, 2022. Phylogenetic tree using neighbor‐joining method and Kimura 80 substitution model. The figure was generated by CLC genomics software](https://i.imgur.com/TKkFNYS.png)|
|**Figure 2:** Kandeel et al., 2022. Phylogenetic tree using neighbor‐joining method and Kimura 80 substitution model. The figure was generated by CLC genomics software|
|![Kandeel et al, 2022. Phylogenetic tree using UPGMA method and Kimura 80 substitution model. The figure was generated by CLC genomics software](https://i.imgur.com/5uWk7b2.png)|
|**Figure 3:** Kandeel et al., 2022. Phylogenetic tree using UPGMA method and Kimura 80 substitution model. The figure was generated by CLC genomics software|

### Data Collection and Software

Data was acquired from the GISAID EpiCov database of genomes collected from patients afflicted by COVID-19 (https://www.gisaid.org).[2] Genomes were selected from the lineages examined by Kandeel et al. The GISAID database is a global collaboration to collect, track, monitor, and share data on emerging epidemics. Due to this collaborative nature, GISAID prohibits its members to share the data itself in any publicly accessible format in order to ensure that its collaborators are properly credited for their contributions to the database. The attribution information and accession IDs are available in the supplementary materials.

In order to better utilize the fixed-rate dendrogram methods, we collected the closest 49 samples to the genomes listed by Kandeel et. al. Due to the overrerpresentation of Japanese sequences in the original dataset, I opted to take samples of my variant lineages from alternative geographic locations for Beta, Delta, and Gamma variant lineages. In order to evaluate the effectiveness of Kandeel et al's original model, I am retaining their original Japanese samples as well. All of the alternative genomes were restricted to samples collected as close to the Japanese genome as possible. A total of 401 genomes were collected from the database.

To perform the alignments, sequences were first entered into Clustal Omega, however such a large dataset proved too 
computationally intense for our hardware. The genomes were then aligned using Nextalign [3], using the Wuhan_Hu_1 genome as 
the reference sequence. It is worth noting that Nextalign is also utilized by the GISAID database for aligning their 
sequences and for tracking the global SARS-CoV-2 phylogeny.
### Phylogenetics
Phylogenetic analysis and tree construction was performed using RAXML Gui [4-8]. RAXML Gui detected that 109 of the initial 
401 sequences were identical, and these were removed before performing the phylogenetic analysis, leaving 292 sequences in 
the final data set. The 292 sequences were then analyzed using RAXML's ML+Rapid Bootstrap method with GTR+GAMMA substitution.
These sequences were bootstrapped 100 times. Additionally, the original 8 sequences analyzed by the authors were analyzed on 
their own, using RAXML's ML+Thorough Bootstrap method. 3 substitution methods were used in order of complexity, which were 
GTR+GAMMA, K80, and JC. The trees generated from the original 8 sequences were bootstrapped 500 times.
________
## Results
____
### Analysis of the Large (292 Genome) Sequences
The Large (292 Genome) dataset produced a phylogenetic tree in which Omicron, Alpha, and Gamma variant form their own branch,
which is in direct conflict with the results obtained by Kandeel et al, 2022 [1] which show Omicron as being a monophyletic 
variant. The phylogenetic trees with variant lineages labeled and color-coded are shown in Figures 4 & 5. Interestingly, this
does reflect the findings of the authors which indicated Alpha being the closest related to Omicron, followed by Gamma.[1] 
There are two possible explanations for this difference, either that the Maximum-Likelihood method was able to capture more 
nuance than the NJM and UPGMA methods utilized by Kandeel et al, or the greater diversity of the genomes provided allowed the
ML methods to have more information which led to the relationship being identified.

|![Figure 4: Phylogenetic Tree](https://i.imgur.com/h2o6r5F.png)|
|:---:|
|**Figure 4:** Phylogenetic Tree of the Large (292) sequence set. Note that Omicron (red), Alpha (Blue), and Gamma (Teal) are located on the same branch, while Beta (Dark Green), MuGH (Magenta), and the European GH490R variant (Purple) cluster with each other. Finally, the American hCoV variant (Orange) is shown as an offshoot of Delta (Lime Green).
|![Figure of rectangular tree with all sequences](https://i.imgur.com/82RXHD5.png)|
|**Figure 5:** Phylogenetic tree of the Large (292) sequence set generated using ML and GTR+GAMMA and support value labels. The tree is displayed with a midpoint root.|

To test these hypotheses and identify the source of this differentiation, the original sequences studied by Kandeel et al 
were analyzed on their own using RAXML Maximum Likelihood with thorough bootstrapping. In order to ensure the most detail was
captured, substitution methods of varying detail were employed in tree construction. These methods were the GTR+GAMMA model,
which takes into account 6 degrees of freedom for transitions and transversions, assigning a separate rate of occurrence for
each possible event; likewise, the K80 has 2 degrees of freedom and JC only has one. There was no significant difference 
identified between the evolutionary distance determined by any of the substitution methods, with the final tree for the 8 
sequences from Kandeel et al. visible in Figure 6. The results of re-analyzing the original 8 sequences shows that the 
structure shown in the Large sequence is maintained even with a smaller sample size and the limited geographic diversity of 
the sequences used by Kandeel et al.

|![Figure 6](https://i.imgur.com/FOllp1Q.jpg)|
|:---:|
|**Figure 6:** 8-sequence Phylogenetic Tree constructed using Maximum Likelihood Methods and GTR+GAMMA. No difference in the bootstrap values or topology was obtained for K-80 or JC substitution models.|

___
## Discussion
___
### Comparison of Trees
A comparison of the trees generated by Kandeel et al, 2022 (Figures 2 & 3)[1] and the novel tree generated in this analysis reveal striking differences. The Omicron variant lineage is shown to be in a distinct clade with the Alpha and Gamma variants, while the GH490R, MuGH, Beta, Delta, and USA hCoV lineages form another. The closeness of Alpha and Gamma with Omicron suggested in Kandeel et al, 2022 [1] is reflected in the Maximum-Likelihood tree, shown in Figure 5. The structure outlined in the Large (292 sequence) tree is maintained even with the original 8 sequences utilized in the Kandeel et al. 2022 article, suggesting that the use of Maximum-Likelihood methods is responsible for the difference between our findings. The Large phylogenetic tree serves as strong support for the relationship identified in the 8-sequence tree, which allows us to contest the hypothesis produced in Kandeel et al, 2022 that Omicron exists as a monophyletic clade [1]. Instead, we assert that Omicron is more closely related to Alpha and Gamma than Alpha and Gamma are to any of the other variants analyzed in the data set. This relationship can be demonstrated at the micro level with solely 8 sequences or at the macro level with 292. 

Two observations in the article published by Kandeel et al. do hold true in our analysis as well. Perhaps the most prominent is the clear association between the U.S. variant of hCoV and the Delta variant, which Kandeel et al likewise identified as being more closely related to each other than any other sequence [1]. This relationship is demonstrated in both of the novel trees generated in this analysis. Additionally, the close relationship noted between Omicron, Alpha, and Gamma is amplified in our study from merely being the closest related to Omicron but in a distinct clade to being included in the same clade as Omicron itself. Due to the heuristic method of tree construction utilized in ML analysis, this relationship was demonstrated as being the most likely out of all possible combinations with the given datasets of 292 sequences and of the 8 sequences used by Kandeel et al. 

### Methodological Considerations
The use of Maximum-Likelihood methods for smaller datasets is demonstrated as being capable of capturing substantial detail 
when working with datasets of large and small sizes. ML methods create a best-fit model of the evolutionary history of a 
given set of sequences, generating a topology which is the best possible match for how the set of sequences originated. The 
use of the GTR+GAMMA method of estimating evolutionary distance alongside Maximum Likelihood methods means that a maximal 
amount of information is considered by the program, given that GTR+GAMMA assumes a set likelihood for every possible 
nucleotide transisition and transversion. ML methods in the past were limited by computing power and memory, but with modern
hardware the analysis of even 400 sequences took less than 24 hours to complete. Methods such as UPGMA and NJM are useful in
their own regard, particularly for processing large datasets where computing power and speed of completion are key concerns.
However, the use of less precise substitution models such as the K80 and JC along with the small sample size can lead to 
detail being lost when these methods are implemented, particularly when dealing with differentiation between variant
lineages rather than separate species.

### Analysis of the Novel Trees
The novel trees (Figures 4, 5, & 6) provide a strong demonstration for the power of maximum likelihood models to estimate 
evolutionary history. Figure 4 more accurately depicts the hierarchical structure of the topography while the rectangular 
tree in Figure 5 is accompanied by support values and the topology reflects the likelihood of support for the placement of 
each branch. The support values for the clustering of the variant lineages are written on the branches themselves, and 
represent the strength of the estimate out of 100 repetitions. Both trees depicted are the best trees generated by RAXML's 
ML+Rapid Bootstrapping method with 100 bootstraps. The preservation of this lineage structure in the tree using only 8 sequences shown in Figure 6 is impressive, as there is only one sequence for each of the variants under exploration - a reduction in scale of nearly 36 times. While the overall relationship between the sequences is maintained, the support values for these relationships are decreased in all areas except for the association between the Delta and USA hCoV variants. This is likely due to the model having insufficient data to support the structure at each of the bootstrapped locations.

## References
1. Kandeel, M. et al. (2022) “Omicron variant genome evolution and phylogenetics,” *Journal of medical virology*, 94(4), pp. 1627–1632.
2. Elbe, S. and Buckland-Merrett, G. (2017) "Data, disease and diplomacy: GISAID’s innovative contribution to global health."" *Global Challenges*, 1:33-46. doi: 10.1002/gch2.1018 PMCID: 31565258
3. Aksamentov, I., Roemer, C., Hodcroft, E. B., & Neher, R. A., (2021). "Nextclade: clade assignment, mutation calling and quality control for viral genomes." *Journal of Open Source Software*, 6(67), 3773, https://doi.org/10.21105/joss.03773
4. Edler, D., J. Klein, A. Antonelli, and D. Silvestro. (2020). "raxmlGUI 2.0: A graphical interface and toolkit for phylogenetic analyses using RAxML." *Methods in Ecology and Evolution*, doi: http://dx.doi.org/10.1111/2041-210X.13512
5. Silvestro, D. and I. Michalak. 2012. "raxmlGUI: a graphical front-end for RAxML." *Organisms Diversity and Evolution* 12:335–337.
6. Stamatakis, A. 2014. "Raxml version 8: a tool for phylogenetic analysis andpost-analysis of large phylogenies." *Bioinformatics* 30:1312–1313
7. Kozlov, A. M., D. Darriba, T. Flouri, B. Morel, and A. Stamatakis. 2019. "RAxML-NG: a fast, scalable and user-friendly tool for maximum likelihood phylogenetic inference." *Bioinformatics* 35:4453–4455.
8.  Darriba, D., D. Posada, A. M. Kozlov, A. Stamatakis, B. Morel, T. Flouri. 2020. "ModelTest-NG: A New and Scalable Tool for the Selection of DNA and Protein Evolutionary Models." *Molecular Biology and Evolution* 37:291–294.

## Supplemental Material
### Attribution for the initial 401 Sequences
**Data Availability**
GISAID Identifier: EPI_SET_230428ao
doi: 10.55876/gis8.230428ao
All genome sequences and associated metadata in this dataset are published in GISAID’s EpiCoV database. To
view the contributors of each individual sequence with details such as accession number, Virus name, Collection
date, Originating Lab and Submitting Lab and the list of Authors, visit 10.55876/gis8.230428ao
**Data Snapshot**
* EPI_SET_230428ao is composed of 401 individual genome sequences.
* The collection dates range from 2021-03-08 to 2022-02-28;
* Data were collected in 21 countries and territories;
* All sequences in this dataset are compared relative to hCoV-19/Wuhan/WIV04/2019 (WIV04), the official reference sequence employed by GISAID (EPI_ISL_402124). Learn more at https://gisaid.org/WIV04.

### Attribution for the Large (292 sequence) Data Set

**Data Availability**
GISAID Identifier: EPI_SET_230501nm
doi: 10.55876/gis8.230501nm
All genome sequences and associated metadata in this dataset are published in GISAID’s EpiCoV database. To
view the contributors of each individual sequence with details such as accession number, Virus name, Collection
date, Originating Lab and Submitting Lab and the list of Authors, visit 10.55876/gis8.230501nm
**Data Snapshot**
* EPI_SET_230501nm is composed of 292 individual genome sequences.
* The collection dates range from 2021-03-08 to 2022-02-28;
* Data were collected in 21 countries and territories;
* All sequences in this dataset are compared relative to hCoV-19/Wuhan/WIV04/2019 (WIV04), the official
reference sequence employed by GISAID (EPI_ISL_402124). Learn more at https://gisaid.org/WIV0
